### ISSUE TITLE
Make it clear in TS-7700 instructions which SD card slot to use

### ISSUE THREAD

**Issue by lorenzo-stoakes** [resin-io/resin-device-types#10]
The TS-7700 has 2 SD card slots, and the one closest to the edge is the one that needs to be used in order to be able to boot. The instructions for this device don't currently specify this and ought to.

**Comment by telphan** [comment]
The TS-7700 also has a hidden dependency on the size of the SD card that it supports. Only SD cards > 8GB are supported by this board. Reason can be found here: https://github.com/resin-os/resin-ts/blob/master/layers/meta-resin-ts/recipes-support/resin-extend-expand/resin-extend-expand/0001-Fix-the-bound-to-a-hard-limit.patch

**Comment by agherzan** [comment]
@telphan good point. Can you please create a separate issue for this? This needs to be addressed better.
